PROJECT

```diff
@@ -6,7 +6,7 @@ resources:
 - group: k8gb
   kind: Gslb
   version: v1beta1
-version: 3-alpha
+version: "3"
```

Comment:
Is it required to be a string?

Reply:
Not sure if it is required, they use it like a string:
 - https://github.com/operator-framework/operator-sdk/pull/4613/files#diff-81aae5fab7d1e80a3b56a8736536bb188e82fe5141d55b22d089c41cd4ec9bfcR71-R82
 - https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.5.0/#project-config-version-3-alpha-must-be-upgraded-to-3

Reply:
I guess using `3` might cause issues with version parsing in osdk toolset, as it's treated as integer type

Reply:
cool, thanks for double checking